packages/infra/lib/api-stack/fhir-converter-connector.ts

```diff
@@ -136,6 +140,8 @@ export function createLambda({
       AXIOS_TIMEOUT_SECONDS: axiosTimeout.toSeconds().toString(),
       ...(config.lambdasSentryDSN ? { SENTRY_DSN: config.lambdasSentryDSN } : {}),
       API_URL: `http://${apiServiceDnsAddress}`,
+      FHIR_SERVER_URL: fhirServerUrl,
+      MEDICAL_DOCUMENTS_BUCKET_NAME: medicalDocumentsBucket.bucketName,
```

Comment:
this doesn't appear to be used in the lambda that I can tell...

Reply:
This is used in the underlying functions that the lambda calls. 
It's used when the attachments are extracted: 
- FHIR Server URL to upload DocumentReferences, so they can be pulled with other documents
- Bucket Name to upload the attachments to S3

Reply:
Hmmm -- I feel like for core code that is executed via lambda, it might make sense to make have the pattern be to accept the env variable as a param then default to the config version (and bail if not set).

I feel like the disconnect between the lambda code and env variables is ripe for bugs, but it's possible we've done this quite a lot. At least for `const region = Config.getAWSRegion();` but it's automatically set by AWS so is relatively safe to call within core.

Reply:
hmmm, i actually like this more. This will also allow me to make the local utils script that we use for FHIR converter tests to not insert stuff on s3/fhir server, unless we want to